### ISSUE TITLE
Example training file

### ISSUE THREAD

**Issue by ntelo007** [hardianlawi/MTL-Homoscedastic-Uncertainty#4]
Could you provide some example files for the training of the network? It seems like it's expecting a CSV file that will provide the full paths to the training images, correct? I would appreciate if you could send me also any other additional but needed file. Thank you for your time.

**Comment by hardianlawi** [comment]
Hi @ntelo007 when I experimented on this, I was using a tabular CSV file, not images data. You can specify the label [here](https://github.com/hardianlawi/MTL-Homoscedastic-Uncertainty/blob/master/scripts/config.py). Also, these scripts were used to solve my specific problem, I would actually suggest you adopting the parts that you need for your problem.

**Comment by ntelo007** [comment]
Could you help me implement the loss function in Keras?

**Comment by tamaraalshekhli** [comment]
@ntelo007 hi, did you figure out how to implement the loss in keras? im facing the same problem
